widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java

```diff
@@ -123,4 +133,12 @@ private void navigateToCall(UiTheme theme, String mediaType) {
         newIntent.putExtra(GliaWidgets.MEDIA_TYPE, mediaType);
         startActivity(newIntent);
     }
+
+    private void navigateToSurvey(UiTheme theme, Survey survey) {
+        Intent newIntent = new Intent(getApplicationContext(), SurveyActivity.class);
+        newIntent.putExtra(GliaWidgets.UI_THEME, theme);
+        newIntent.putExtra(GliaWidgets.SURVEY, (Parcelable) survey);
```

Comment:
Does it need to be casted?

Reply:
It's questionable why both `Serializable` and `Parcelable` are needed, but it's not relevant in this PR scope. So I resolve this comment.